blackbird_python/blackbird/blackbirdLexer.py

```diff
@@ -1,10 +1,11 @@
-# Generated from blackbird.g4 by ANTLR 4.7.1
+# Generated from src/blackbird.g4 by ANTLR 4.8
```

Comment:
Out of curiosity, does this affect the version of `antlr4-python3-runtime` we are currently pinning in setup.py and requirements.txt? We're currently pinning `4.7.1`, but this might require us to upgrade the requirements

Reply:
I updated this in the requirements since I happened to use `antlr4-python3-runtime=4.8` while generating the files. Could there be any other issues with this that I'm missing?

Reply:
Nope, should be fine! I shouldn't have commented until I read the PR to the end 😆